puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-hammer-nightly-systest.yaml

```diff
@@ -0,0 +1,36 @@
+- job:
+    name: foreman-hammer-nightly-systest
+    node: 'el && ipv6'
+    project-type: matrix
+    logrotate:
+      daysToKeep: 8
+      maxToKeep: 30
+    scm:
+      - forklift
+    triggers:
+      - timed: 'H 5 * * 1,4'
+    axes:
+      - axis:
+          type: user-defined
+          name: os
+          values:
+            - centos7
+    builders:
+      - systest:
+          repo: nightly
+          repo_environment: release
+          os: ''
```

Comment:
Unfortunately with how builder macros work, yes. Though I've noticed that `os` comes from the matrix in every one of these systest jobs, so it could simply be removed completely.

Reply:
`os` and `{os}` removed